app.rb

```diff
@@ -0,0 +1,89 @@
+require_relative('./book')
+require_relative('./student')
+require_relative('./teacher')
+require_relative('./rentals')
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def intialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def books_list
+    books.each { |book| puts `Title: #{book.title}, Author: #{book.author}` }
+  end
+
+  def people_list
+    people.each { |person| puts `Name: #{person.name} ID: #{person.id} Age: #{person.age}` }
+  end
```

Comment:
- [ ] - When I try to list the people or books, it shows the error below. Please, try to handle the case when there are no people or books, your app should show no errors.
<img width="634" alt="image" src="https://user-images.githubusercontent.com/77190203/181914392-6b91e75d-50cc-4562-8ac7-698a9791e23d.png">
<img width="623" alt="image" src="https://user-images.githubusercontent.com/77190203/181914460-19839b7b-efd5-48cd-9633-c4cd34d3656c.png">